### ISSUE TITLE
Is it possible to implement iCarousel inside the UITableviewcell?

### ISSUE THREAD

**Issue by mykrr** [nicklockwood/iCarousel#420]
In Table view cell, i have added a view inside table view cell and added the subview of "iCarousel view". 
While dragging the iCarousel view(using pan gesture), there is no action performed.

**Comment by adamszeptycki** [comment]
Yes it is possible. You had to do something wrong. I implemented such setup in app I'm currently working on and it works great.

**Comment by parinshah30** [comment]
@mykrr @adamszeptycki do you have an example of this
